docs/src/main/tut/pull_request.md

```diff
@@ -128,6 +130,57 @@ createPullRequestIssue.exec[cats.Id, HttpResponse[String]]() match {
 
 See [the API doc](https://developer.github.com/v3/pulls/#create-a-pull-request) for full reference.
 
```

Comment:
Since Reviews are a "sub-API" of the pull request API, we usually do subsections, so you'll end up with:

## Pull request 

### List pull requests

...

## Reviews

### List reviews

### Get an individual review